src/core/core.scale.js

```diff
@@ -716,7 +716,7 @@ module.exports = Element.extend({
 	},
 
 	// Actually draw the scale on the canvas
-	// @param {rectangle} chartArea : the area of the chart to draw full grid lines on
+	// @param {object} chartArea - the area of the chart to draw full grid lines on
```

Comment:
And, this must be surrounded by `/** ... */`.

Reply:
done